temboardagent/httpd.py

```diff
@@ -333,5 +333,9 @@ def default(self, obj):
     defaults = list(json.dumps.func_defaults)
     defaults[4] = JSONEncoder
     json.dumps.func_defaults = tuple(defaults)
+elif sys.version_info < (3, 6):
+    defaults = list(json.dumps.__defaults__)
+    defaults[4] = JSONEncoder
+    json.dumps.__defaults__ = tuple(defaults)
```

Comment:
I wonder if we couldn't update calls to `json.dumps()` to pass our custom JSONEncoder rather than monkeypatching the function here in a quite ugly manner. AFAICT, this should be done only once in this file in `response()` method.